The CI builds for cli-hooks package failure

The CI builds for cli-hooks package started failing on August 19 or so although there is no change since the last revision. Perhaps, dependency version issue or something like that? 

@zimeg I assigned you for now because you've been working on the package but please feel free to re-assign or just remove it if it's tno relevant or you don't have the bandwidth for it.

```
> @slack/cli-hooks@1.1.1 prelint
> tsc --noemit --module es2022 --project ./jsconfig.json

Error: node_modules/punycode/punycode.js(42,23): error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ overflow: string; 'not-basic': string; 'invalid-input': string; }'.
  No index signature with a parameter of type 'string' was found on type '{ overflow: string; 'not-basic': string; 'invalid-input': string; }'.
Error: node_modules/punycode/punycode.js(48,12): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
Error: node_modules/punycode/punycode.js(51,14): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
Error: node_modules/punycode/punycode.js(94,9): error TS1003: Identifier expected.
Error: node_modules/punycode/punycode.js(95,9): error TS1003: Identifier expected.
Error: node_modules/punycode/punycode.js(99,14): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
Error: node_modules/punycode/punycode.js(127,9): error TS1003: Identifier expected.
Error: node_modules/punycode/punycode.js(130,12): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
Error: node_modules/punycode/punycode.js(137,9): error TS1003: Identifier expected.
Error: node_modules/punycode/punycode.js(159,9): error TS1003: Identifier expected.
Error: node_modules/punycode/punycode.js(168,38): error TS7006: Parameter 'flag' implicitly has an 'any' type.
Error: node_modules/punycode/punycode.js(171,27): error TS2363: The right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
Error: node_modules/punycode/punycode.js(171,43): error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
Error: node_modules/punycode/punycode.js(179,24): error TS7006: Parameter 'delta' implicitly has an 'any' type.
Error: node_modules/punycode/punycode.js(179,31): error TS7006: Parameter 'numPoints' implicitly has an 'any' type.
Error: node_modules/punycode/punycode.js(179,42): error TS7006: Parameter 'firstTime' implicitly has an 'any' type.
Error: node_modules/punycode/punycode.js(294,2): error TS2322: Type 'any[]' is not assignable to type 'string'.
Error: node_modules/punycode/punycode.js(306,7): error TS2365: Operator '<' cannot be applied to types 'string' and 'number'.
Error: node_modules/punycode/punycode.js(307,35): error TS2[34](https://github.com/slackapi/node-slack-sdk/actions/runs/10448756253/job/28929779489#step:6:35)5: Argument of type 'string' is not assignable to parameter of type 'number'.
Error: node_modules/punycode/punycode.js(329,8): error TS2365: Operator '>=' cannot be applied to types 'string' and 'number'.
Error: node_modules/punycode/punycode.js(329,29): error TS2365: Operator '<' cannot be applied to types 'string' and 'number'.
Error: node_modules/punycode/punycode.js(330,5): error TS2322: Type 'string' is not assignable to type 'number'.
Error: node_modules/punycode/punycode.js(345,8): error TS2365: Operator '<' cannot be applied to types 'string' and 'number'.
Error: node_modules/punycode/punycode.js(348,8): error TS2367: This comparison appears to be unintentional because the types 'string' and 'number' have no overlap.
Error: node_modules/punycode/punycode.js(390,[35](https://github.com/slackapi/node-slack-sdk/actions/runs/10448756253/job/28929779489#step:6:36)): error TS7006: Parameter 'string' implicitly has an 'any' type.
Error: node_modules/punycode/punycode.js(409,35): error TS7006: Parameter 'string' implicitly has an 'any' type.
Error: node_modules/punycode/punycode.js(429,10): error TS1003: Identifier expected.
Error: Process completed with exit code 2.
```
https://github.com/slackapi/node-slack-sdk/actions/runs/10448756253/job/28929779489

--- a/packages/cli-hooks/package.json
+++ b/packages/cli-hooks/package.json
@@ -37,7 +37,7 @@
   "scripts": {
     "prebuild": "shx rm -rf ./coverage",
     "build": "shx chmod +x src/*.js",
-    "prelint": "tsc --noemit --module es2022 --project ./jsconfig.json",
+    "prelint": "tsc --noemit --module es2022 --maxNodeModuleJsDepth 0 --project ./jsconfig.json",
     "lint": "eslint --ext .js src",
     "pretest": "npm run lint -- --fix",
     "test": "c8 mocha src/*.spec.js"

